plugins/jdg66/src/main/java/org/radargun/service/JDG66EmbeddedQueryable.java

```diff
@@ -0,0 +1,94 @@
+package org.radargun.service;
+
+import org.infinispan.query.Search;
+import org.infinispan.query.dsl.Expression;
+import org.infinispan.query.dsl.FilterConditionEndContext;
+import org.infinispan.query.dsl.QueryFactory;
+
+/**
+ * Adds support for aggregations.
+ *
+ * @author Jakub Markos &lt;jmarkos@redhat.com&gt;
+ */
+public class JDG66EmbeddedQueryable extends Infinispan70EmbeddedQueryable {
```

Comment:
There is, but it requires Java 8. I thought that the 2.x branch was meant to stay on 7, am I wrong?